app/javascript/angular/tests/_fixed_sessions.test.js

```diff
@@ -360,6 +361,20 @@ test('drawSessionsInLocation draws colorcoded marker for currently streaming ses
   t.end();
 });
 
+test('drawSessionsInLocation calls map.clusterMarkers for currently streaming sessions when sensor selected', t => {
+  const clusterMarkers = sinon.spy();
+  const map = { clusterMarkers }
+  const sensors = { anySelected: () => true, selectedSensorName: () => "sensorName"};
+  const data = buildData({ location: { streaming: true } });
```

Comment:
Must be `streaming: true` for the test to pass?

Reply:
Yes, we only want to cluster for currently streaming sessions. In other cases, the method returns early.